### PR TITLE
Replace 'one-off' by 'bespoke'

### PR DESCRIPTION
**Pull request by MartinGuindon**
Since the documentation has changed the terminology from data test to bespoke test and uses bespoke at the main heading in the main Tests docs (https://docs.getdbt.com/docs/building-a-dbt-project/tests#bespoke-tests), it is confusing to use `one-off test` here. I'm therefore proposing to update this to use `Bespoke` here as well.

## Pre-release docs
Is this change related to an unreleased version of dbt?
- [ ] Yes: please
    - update the base branch to `next`
    - add Changelog components: `<Changelog>[New/Changed] in v0.x.0</Changelog>`
    - add links to the "New and changed documentation" section of the latest [Migration Guide](../website/docs/docs/guides/migration-guide)
- [X] No: please ensure the base branch is `current`
- [ ] Unsure: we'll let you know!

## Checklist
If you added new pages (delete if not applicable):
- [ ] The page has been added to `website/sidebars.js`
- [ ] The new page has a unique filename

If you removed existing pages (delete if not applicable):
- [ ] The page has been removed from `website/sidebars.js`
- [ ] An entry has been added to `_redirects`

### CODE REVIEW

**Comment by jtcohen6** [comment]
@MartinGuindon The truth is that I haven't yet decided on the final name for this test type, whether `bespoke` or `one_off` (or `specific` or `single_use`). There's some discussion about this over in https://github.com/dbt-labs/dbt/issues/3234, since we still need to change these for real in the Core codebase.

**Comment by MartinGuindon** [comment]
> @MartinGuindon The truth is that I haven't yet decided on the final name for this test type, whether `bespoke` or `one_off` (or `specific` or `single_use`). There's some discussion about this over in [dbt-labs/dbt#3234](https://github.com/dbt-labs/dbt/issues/3234), since we still need to change these for real in the Core codebase.

Gotcha. Replied with my thoughts on the issue. :)

File: website/docs/reference/resource-configs/store_failures.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ This logic is encoded in the [`should_store_failures()`](https://github.com/fish
   defaultValue="specific"
   values={[
     { label: 'Specific test', value: 'specific', },
-    { label: 'One-off test', value: 'one_off', },
+    { label: 'Bespoke test', value: 'bespoke', },
```
Comment (runleonarun) [inline website/docs/reference/resource-configs/store_failures.md]:
```suggestion
    { label: 'Singular test', value: 'singular', },
```

File: website/docs/reference/resource-configs/store_failures.md [hunk 2]
```diff
@@ -53,9 +53,9 @@ models:
 
 </TabItem>
 
-<TabItem value="one_off">
+<TabItem value="bespoke">
```
Comment (runleonarun) [inline website/docs/reference/resource-configs/store_failures.md]:
```suggestion
<TabItem value="singular">
```

File: website/docs/reference/resource-configs/store_failures.md [hunk 3]
```diff
@@ -53,9 +53,9 @@ models:
...
 
-Configure a one-off (data) test:
+Configure a bespoke (data) test:
```
Comment (runleonarun) [inline website/docs/reference/resource-configs/store_failures.md]:
```suggestion
Configure a singular (data) test:
```

**Review by runleonarun — CHANGES_REQUESTED** [review]
Thank you for this edit! Looks like this was changed to singular so I made that change.

**Review by runleonarun — APPROVED** [review]
LGTM!

**Pull request merged**